apps/web-neutron/src/components/layout/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { CSSObject } from '@emotion/react';
+import { makeStyles } from 'tss-react/mui';
+
+const useStyles = makeStyles()((theme) => ({
+  root: {
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      minHeight: '100vh',
+    },
+  },
+  contentWrapper: {
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flex: 1,
+    },
+  },
+  footer: {
+    [theme.breakpoints.up('lg')]: {
+      position: 'relative',
+      zIndex: 1299,
+    },
+  },
+  appBarPlaceholder: {
+    ...(theme.mixins.toolbar as CSSObject),
```

Comment:
When spreading the same object, can avoid it which will be better for performance:
```typescript
appBarPlaceholder: theme.mixins.toolbar as CSSObject,
```